packages/react-components/react-virtualizer/stories/Virtualizer/Default.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Virtualizer } from '@fluentui/react-components/unstable';
+import { makeStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    overflowAnchor: 'none',
+    overflowY: 'auto',
+    width: '100%',
+    height: '100%',
+    maxHeight: '750px',
+  },
+  child: {
+    height: '100px',
+    lineHeight: '100px',
+    width: '100%',
+  },
+});
+
+export const Default = () => {
+  const styles = useStyles();
+  const childLength = 1000;
+  const childList = React.useMemo(() => {
+    const nodeList: React.ReactNode[] = [];
+    for (let i = 0; i < childLength; i++) {
+      nodeList.push(<span key={`test-virtualizer-child-${i}`} className={styles.child}>{`Node-${i}`}</span>);
+    }
+    return nodeList;
+  }, [styles.child]);
+
+  return (
+    <div className={styles.container}>
```

Comment:
might want to check up with @smhigley  that the examples are accessible, they will be on our public docsite

Reply:
Chatted w/ @Mitch-At-Work about adding list semantics to the examples. I'd also add it'd be good to give each example list an accname through `aria-label` to make it easier to jump through them on the example page (though it's not super important, since there will be headings in the storybook docs)